tinygrad/renderer/cstyle.py

```diff
@@ -75,6 +75,7 @@ class CStyleLanguage(Renderer):
   extra_args: list[str] = []
   float4: Optional[str] = None
   type_map: dict[DType, str] = {}
+  invalid_dtype: dict[DType, None] = {}
```

Comment:
Why not just a list?

Reply:
I think it could be a list as well, other parts of the code base use a dict for such lookup purpose so I just followed the style

Reply:
Other parts of the code base use `dict[sth, None]` to preserve order, I guess set is correct.